seatunnel-connectors-v2/connector-kudu/src/main/java/org/apache/seatunnel/connectors/seatunnel/kudu/kuduclient/KuduInputFormat.java

```diff
@@ -136,7 +138,8 @@ public SeaTunnelRowType getSeaTunnelRowType(List<ColumnSchema> columnSchemaList)
             }
         } catch (Exception e) {
             log.warn("get row type info exception.", e);
-            throw new PrepareFailException("kudu", PluginType.SOURCE, ExceptionUtils.getMessage(e));
+            throw new KuduConnectorException(KuduConnectorErrorCode.GET_ROW_TYPE_INFO_FAILD, String.format("PluginName: %s, PluginType: %s, Message: %s",
```

Comment:
CommonErrorCode.TABLE_SCHEMA_GET_FAILED is better, remove this unclear error code.